samples/azure-quantum/dynamic-error-correction/DynamicErrorCorrection.ipynb

```diff
@@ -0,0 +1,369 @@
+        "In this program you will learn to:\n",
+        "1. **Represent a logical qubit** through a register.\n",
+        "2. **Perfom a dynamic error correction**. <br/>The error correction scheme is trivial. It assumes that a single bit-flip error has occurred and does not consider the possibility of errors occurring during the measurement of the ancillary qubits."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "%azure.connect \"/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.Quantum/Workspaces/workspaceName\" location=\"westus\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "open Microsoft.Quantum.Intrinsic;\n",
+        "open Microsoft.Quantum.Math;\n",
+        "open Microsoft.Quantum.Measurement;"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## 1. Encode a qubit register\n",
+        "The qubit register now represents a logical qubit."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "operation Encode(register : Qubit[]) : Unit is Adj {\n",
+        "    CNOT(register[0], register[1]);\n",
+        "    CNOT(register[0], register[2]);\n",
+        "}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## 2. Set value for each register\n",
+        "Arbitrarily, we set the register value to One."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "operation LogicalX(register : Qubit[]) : Unit {\n",
+        "    for qubit in register {\n",
+        "        X(qubit);\n",
+        "    }\n",
+        "}"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## 3. Perform arbitrary operation\n",
+        "The Rx gate has a period of 4 * pi. Applying ```PI() * 0.5``` 8 times should return the qubit to its original state.\n",
+        "<br/>As we are manipulating physical qubits, we are however expecting some errors to be introduced."
```

Comment:
I'm not sure I understand this statement. Does this only apply to hardware and noise?

Reply:
@swernli could you comment?

Reply:
I think @cesarzc would be better suited to evaluate this one.

Reply:
This is not exclusive to hardware and noise. This operation is used to test the error correction code, so we construct an identity operation using rotations around the x-axis.

I've updated the description and hopefully now is clearer.